governance/tech-lead-proposal-process.md

```diff
@@ -0,0 +1,107 @@
+
+### Nominees by community
+
+Community nominees should adhere to the technical lead nomination requirements
+described below.
+
+The co-chairs' reserve the right to endorse a nominee by the community based on:
+
+* Gender diversity
+* Company diversity
+* Geo diversity
+* Amount of nominations from different companies, people, regions for a single
+  individual
+* Existing body of work
+* Justification provided during nomination
+
+#### Community timeline
+
+Community nominations will to run for at least 1 month with a maximum up to 2
+months. Upon close of nominations a decision will be made by the co-chairs
+within the next month in order conform to the TOC process.
+
+### Nominees by co-chairs
+
+Nominees by co-chairs adhere to the technical lead requirements described below.
+Co-chairs are responsible for documenting statistic information of all nominee
+types for endorsement and presentation to the TOC.
+
+#### Endorsement of community nominees
+
+Co-chairs are responsible for providing endorsement of a community nominee to
+show concurrence and support.
+
+If no nominees are submitted by the community either as a result of lack of
+participation, lack of requirements, or from co-chair specific selection
+(non-community nomination process), the co-chairs must endeavor to nominate an
+individual(s) that meet as many of the requirements as exist.
+
+## Technical Lead nomination requirements
+
+In order to verify nominees by the community or by co-chairs have experience
+with the group and are actively contributing in a meaningful way, the following
+requirements must be met in order for an individual to be nominated.
+
+If a given nominee does not meet all the requirements, the co-chairs will still
+review the nomination and justification to determine if enough intent and
+commitment is present to move forward with an endorsement.
+
+Nominees must:
+<!--markdownlint-disable MD005 MD007 -->
+
+* have been project or review lead on at least one project or security assessment/review
+* be the author of at least one pull request against the Security TAG repository
+* be active in the community within the last 6 months prior to their nomination.
+ * active - Attending at least 1 Security TAG Regular Meeting a month
+ * engaging in community chat via slack or lists
+ * commenting on PRs and issues to drive suggestions to proposals, define scope,
+   resolve clarity issues, etc.
+* agree to the nomination
+<!-- markdownlint-enable MD005 MD007 -->
+
+Nominations must:
+
+* come from someone other than the nominee
+* have justification from the nominator regarding the nominee's capabilities
+
+### Justification for nominees
+
+To ensure nominations are thoughtful and supported, in addition to the above
+requirements, a justification must enumerate on the nominee's:
+
+* STAG Community involvement,
+* leadership skills,
+* conflict resolution (technical and non-technical),
+* mentorship, willingness to learn and expand existing technical knowledge,
```

Comment:
Think mentorship and willingness to learn/technical knowledge probably are 2 separate bullet points here